rollup.config.js

```diff
@@ -72,16 +79,35 @@ const prodUmdConfig = Object.assign({}, umdConfig, {
   ),
 })
 
-const webConfig = Object.assign({}, configBase, {
+const serverConfig = Object.assign({}, configBase, {
+  external: configBase.external.concat('stream'),
+  output: [
+    { file: 'dist/styled-components.es.js', format: 'es' },
+    Object.assign({}, cjs, { file: 'dist/styled-components.cjs.js' }),
+  ],
+  plugins: configBase.plugins.concat(
+    replace({
+      __SERVER__: JSON.stringify(true),
+    })
+  ),
+})
+
+const browserConfig = Object.assign({}, configBase, {
   output: [
-    { file: pkg.module, format: 'es' },
-    Object.assign({}, cjs, { file: pkg.main }),
+    { file: 'dist/styled-components.browser.es.js', format: 'es' },
```

Comment:
you could use `pkg.browser[pkg.module]` here

Reply:
I ended up not using these because they made scanning for a particular build in the file more difficult.